src/main/resources/documentation/service-description.html

```diff
@@ -11,63 +12,77 @@ <h2 id="data-provided">Data provided</h2>
     <a href="https://api-dev.prison.service.justice.gov.uk/swagger-ui/index.html">prison-api</a>
     might be a better service to use.
     One use case this might be appropriate for is if you need up-to-date data at the point a NOMIS prisoner event
-    is received; this service also updates its records as a result of NOMIS prisoner events, so you might encounter a race
+    is received; this service also updates its records as a result of NOMIS prisoner events, so you might encounter a
+    race
```

Comment:
Are we better off not putting in any line breaks here so the web browser sorts it out? Or does that look a bit funky in real life?

Reply:
intellij doesn't automatically wrap though, hence formatting.  Won't make a difference to the web browser output anyway.  Agree though that the formatting now (especially that line) does look rubbish, hadn't spotted that 🤦